corehq/apps/cloudcare/dbaccessors.py

```diff
@@ -19,16 +19,5 @@ def get_application_access_for_domain(domain):
 
 
 def get_cloudcare_apps(domain):
-    result = ApplicationBase.get_db().view(
-        'app_manager/applications_brief',
-        startkey=[domain],
-        endkey=[domain, {}]
-    )
-    app_docs = [row['value'] for row in result]
-    # Note: even though cloudcare_enabled is in the value emitted by
-    # the view, couch will not include it in the emitted value if
-    # it's undefined.
-    return [
-        app for app in app_docs
-        if app['doc_type'] == 'Application' and app.get('cloudcare_enabled', False)
-    ]
+    apps = get_brief_apps_in_domain(domain, include_remote=False)
+    return [app for app in apps if app.cloudcare_enabled]
```

Comment:
dict access still works on wrapped models so it should all be fine. I did test a few workflows and didn't get any errors.